### PR TITLE
Update service to reflect current credential format

### LINKED ISSUE #1: uri -> url
GP credentials return a field `credentials.url` not `credentials.uri`. (This changed between the experimental and beta version of the service.) 
The module should accept `credentials.url || credentials.uri` and show an error if falsy.

### PR DESCRIPTION
**Pull request by srl295**
Fixes #1

### CODE REVIEW

File: src/gp-angular.js [hunk 1]
```diff
@@ -263,7 +264,7 @@ module.provider('GlobalizationPipelineService', [ function() {
                   }
                   languageDeferred.resolve(lang);
               }, function errorCallback(error) {
-                  var result = '[normalizeLanguage] ERR: ' + error.toString();
+                  var result = '[normalizeLanguage] ERR: ' + JSON.stringify(error);
```
Comment (srl295) [inline src/gp-angular.js]: this is just to output a real message instead of "[Object]"

File: src/gp-angular.js [hunk 2]
```diff
@@ -116,7 +116,8 @@ module.provider('GlobalizationPipelineService', [ function() {
          */
         function getBundleInfo(bundleId) {
             var restDefer = $q.defer();
-            var url = gp_config.credentials.uri + '/rest/' + gp_config.credentials.instanceId + '/' + gpVersion + '/bundles' + '/' + bundleId;
+            var url = (gp_config.credentials.url || gp_config.credentials.uri)
```
Comment (srl295) [inline src/gp-angular.js]: support old form (uri) also url.

But the url DOES include /rest

File: src/gp-angular.js [hunk 3]
```diff
@@ -374,7 +375,8 @@ module.provider('GlobalizationPipelineService', [ function() {
 
             getCache().addBundle(gpBundleKey, languageId, restDefer.promise);
 
-            var url = gp_config.credentials.uri + '/rest' + '/' + gpBundleKey + '/' + languageId;
+            var url = (gp_config.credentials.url || gp_config.credentials.uri)
```
Comment (srl295) [inline src/gp-angular.js]: see above

**Comment by srl295** [comment]
@yumaoka please review

**Comment by yumaoka** [comment]
LGTM

**Pull request merged**